Source/WebCore/Modules/WebGPU/Implementation/WebGPUCompositorIntegrationImpl.cpp

```diff
@@ -62,8 +62,10 @@ void CompositorIntegrationImpl::prepareForDisplay(uint32_t frameIndex, Completio
 void CompositorIntegrationImpl::updateContentsHeadroom(float headroom)
 {
 #if HAVE(SUPPORT_HDR_DISPLAY)
-    for (auto& ioSurface : m_renderBuffers)
+    for (auto& ioSurface : m_renderBuffers) {
+        ALWAYS_LOG_WITH_STREAM(stream << "__GS__ CompositorIntegrationImpl[" << this << "]::updateContentsHeadroom(" << headroom << ") -> ioSurface[" << &ioSurface << "]->setContentEDRHeadroom(" << headroom << ")");
```

Comment:
There's an 'HDR' log channel in WebCore now, if you want to make some of this logging more-permanent.